Answers.md

```diff
@@ -1,3 +1,29 @@
-### Your responses to the short answer questions should be laid out here using markdown.
+# Describe the differences between a Docker container and a virtual machine. What makes a container more aptly-suited for portability?
 
-For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+A Docker container is an image that contains everything needed to run some software whereas a virtual machine is an entire os running on another os. Both Docker and virtual machines isolate themselves from the OS they are running on, but docker itself is just software being executed by the OS.
+
+# Given the command `docker run -p 49160:8080 -d <your_docker_username>/<your_docker_image_name>`, what does `49160:8080` specify?
+
+You are binding port 49160 of the host machine to the docker machine port 8080. So you are essentially forwarding traffic from port 49160 to the docker machine port 8080
+
+# What is the main purpose of using a "container orchestration platform" such as Kubernetes or Docker Swarm?
+
+Container orchestration platforms allow you to manage clusters of docker images to scale your software to meet demands.
```

Comment:
Also because all the different parts of a modern web application need to each be contained within its own container. An orchestration platform handles the coordination between all these multiple containers.